src/flashbar/collapsible-flashbar.tsx

```diff
@@ -48,7 +51,7 @@ export default function CollapsibleFlashbar({ items, ...restProps }: FlashbarPro
     setInitialAnimationState(rects);
   }, [getElementsToAnimate]);
 
-  const { baseProps, breakpoint, isReducedMotion, isVisualRefresh, mergedRef, ref } = useFlashbar({
+  const { baseProps, breakpoint, isReducedMotion, mergedRef, ref } = useFlashbar({
```

Comment:
Moved `isVisualRefresh` from the hook directly into this file to make DCE work for ALWAYS_VISUAL_REFRESH mode